packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -97,6 +97,11 @@ const useRootStyles = makeStyles({
       // This is if the user clicks the field again while it's already focused
       borderBottomColor: tokens.colorCompoundBrandStrokePressed,
     },
+    ':focus-within': {
```

Comment:
Makes me wonder if we'll need a shorthand method for outline

Reply:
Filed #21231